src/index.html

```diff
@@ -1,13 +1,392 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link
+     rel="icon"
+     href="images/icons/favicon-36px.png"
+     sizes="36x36 64x64"
+     type="image/png"
+     >
+    <title>Kickstarter</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="body">
+    <div class="body__container">
+      <header class="header" id="header">
+        <div class="header__container">
+          <div class="top-bar">
+            <a href="#main">
+              <img
+                src="./images/icons/logo.svg"
+                alt="crazybaby"
+                class="header__logo"
+              >
+            </a>
+            <div class="top-bar__block">
+              <div class="top-bar__language">
+                <a href="#" class="top-bar__link"><p class="top-bar__subtitle">SPECS</p></a>
+                <a href="#" class="top-bar__link"><p class="top-bar__subtitle"><span class="top-bar__span">EN</span>/ UA</p></a>
+              </div>
+              <div class="top-bar__button">
+                <a
+                  href="https://github.com/LukashevychSergey"
+                  class="top-bar__button--link"
+                  target="_blank"
+                >buy</a>
```

Comment:
Need to fix code style everywhere
```suggestion
                <a
                  href="https://github.com/LukashevychSergey"
                  class="top-bar__button--link"
                  target="_blank"
                >
                  buy
                </a>
```